examples/advanced/quantum_volume.ipynb

```diff
@@ -0,0 +1,109 @@
+{
+ "cells": [
```

Comment:
I'd split this up into >1 cells to be less intimidating for the reader

Reply:
Done.